packages/core/src/agent/getOutboundMessageContext.ts

```diff
@@ -67,6 +91,14 @@ export async function getOutboundMessageContext(
     )
   }
 
+  if (
+    !(message instanceof DidCommV1Message) ||
```

Comment:
I think we can invert all these statements?

```suggestion
    message instanceof DidCommV2Message
```